### DISCUSSION TITLE
Encountering the Mysterious 546 Error: Seeking Insights on Supabase Edge Functions & Stripe Sync

### DISCUSSION THREAD

**Discussion by kola-white** [supabase/supabase#17899] [Questions]
**Hello Supabase Community! 🚀**

First off, big thanks to the creators and contributors of Supabase for developing such a promising platform. I'm currently on a journey of integrating Supabase into my Next.js project based on the Vercel subscriptions-starter template. My goal? To have a harmonious synchronization between Supabase and my Stripe integrations for managing subscriptions and payments.

However, I've hit a bit of a snag. 🤔

While diving into Edge Functions, which I was genuinely excited about, I've encountered a mysterious HTTP status code `546` during my tests. Now, this seems to be an enigma because I couldn't find any mention of it in the [official Supabase docs](https://supabase.com/docs/guides/platform/http-status-codes).

I've tried to follow along with the tutorials and even the simple "Hello Functions" example, but after deploying it around 10 times using the Supabase CLI, I'm still facing this roadblock. The real puzzler for me is that my usage is way under the limits – I'm on the free plan, and the data involved is less than 1MB, with a virtually instant boot time. 

The big picture? I'm hoping to seamlessly process and store data in sync with my Stripe integrations. But for weeks now, my Supabase tables seem to be missing any Stripe syncs.

So here's my humble request to this fantastic community: Has anyone encountered this mysterious `546` error or can shed some light on it? I'd greatly appreciate any guidance, insights, or shared experiences.

Thank you in advance for taking the time to read this and for any assistance you can provide. Here's to building and learning together! 🌟

**Comment by sansavision** [comment]
Experiencing the same issue! cpu time returns null in the logs and no no records of memory usage, I'm just trying to create a client and delete a user, that's it! after 20ms it fails
